pkg/traces/remotewriteexporter/exporter.go

```diff
@@ -161,25 +160,17 @@ func (e *remoteWriteExporter) handleHistogramIntDataPoints(app storage.Appender,
 }
 
 func (e *remoteWriteExporter) processScalarMetric(app storage.Appender, m pdata.Metric) error {
-	switch m.DataType() {
-	case pdata.MetricDataTypeSum:
-		dataPoints := m.Sum().DataPoints()
-		if err := e.handleScalarIntDataPoints(app, m.Name(), counterSuffix, dataPoints); err != nil {
-			return err
-		}
-	case pdata.MetricDataTypeGauge:
-		dataPoints := m.Gauge().DataPoints()
-		if err := e.handleScalarIntDataPoints(app, m.Name(), noSuffix, dataPoints); err != nil {
-			return err
-		}
+	dataPoints := m.Gauge().DataPoints()
```

Comment:
The data type check is necessary to correctly get the datapoints. While maybe spanmetrics only generated gauges (unsure), it's safer to keep it.

Reply:
Ah, my bad, sorry, missed that they where different, thought the only diff where the prefix!

Reply:
Fixed! Thanks for catching that!